src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -340,6 +352,11 @@ class DynamicTerminationFilter::CallData {
     calld->lb_call_->StartTransportStreamOpBatch(batch);
   }
 
+  static void PreCancel(grpc_call_element* elem, grpc_error_handle error) {
+    auto* calld = static_cast<CallData*>(elem->call_data);
+    calld->lb_call_->PreCancel(error);
```

Comment:
Yes.  `SetPollent()` will always have been called before this is called, because the dynamic filter stack does that on call creation:

https://github.com/grpc/grpc/blob/266d0f7c052bdb686e93525c587e2b9513b35fa7/src/core/ext/filters/client_channel/dynamic_filters.cc#L59